### PR TITLE
build(scss): remove node-sass and any use of scss in components

### PR DESCRIPTION
**Pull request by dcwither**
### Summary:
No more scss, which most importantly means no more node-sass!

### Test Plan:
`npm start` + CI. If someone else can pull down to test it out (you may need to clear your storybook cache with `rm -rf packages/components/node_modules/.cache/storybook` (I should maybe add a note about this in our `README.md`.

### CODE REVIEW

File: packages/components/docs/token-table.tsx [hunk 1]
```diff
@@ -11,12 +11,26 @@ type TokenTableProps = {
   renderExample: (name: string, value: string) => JSX.Element;
 };
 
+const StyledTable = styled.table`
```
Comment (dcwither) [inline packages/components/docs/token-table.tsx:14]: very open to naming suggestions - not a super fan of this name - I could also do a css block instead.

Comment (ahuth) [reply packages/components/docs/token-table.tsx:14]: Seems reasonable. Could also be something like `TokensTable`

**Review by ahuth — APPROVED** [review]
Hell yes 🤘

File: packages/components/docs/token-table.tsx (hunk 1 above)
Comment (dcwither) [reply packages/components/docs/token-table.tsx:14]: The problem was the name overlap with `TokenTable` so I'd rather not have it be a deceptively similar name.

I'll merge for now, and we can discuss the naming conventions for this sort of thing later.

**Pull request merged**